TTLocalizer: Running in language: english
from toontown.toonbase.TTLocalizerEnglish import *
OTPLocalizer: Running in language: english
from otp.otpbase.OTPLocalizerEnglish import *
Preloading suits...
Preloading Toon heads...
Preloading avatars...
Traceback (most recent call last):
  File "D:\Documents\ToontownSourcecode\Toontown\toontown\toonbase\ClientStart.py", line 234, in loadScripts
    execfile('mods/'+scriptName,globals())
  File "mods/Compilation.txt", line 206, in <module>
    toonAnywhere = DirectButton(text = ("Enable Toonup anywhere", "ENABLE TOONUP ANYWHERE", "enable toonup anywhere", "enable toonup anywhere"), scale=.06, pos = (-1,-0,-.3), command=ToonupAnywhere.toonUpSetUp)
NameError: name 'ToonupAnywhere' is not defined

phase_3.5/models/gui/friendslist_gui.bam
['Horiz_Arrow_UP', 'Horiz_Arrow_DN', 'Horiz_Arrow_Rllvr']
phase_3.5/models/gui/friendslist_gui.bam
['Horiz_Arrow_UP', 'Horiz_Arrow_DN', 'Horiz_Arrow_Rllvr']
phase_3.5/models/gui/friendslist_gui.bam
['Horiz_Arrow_UP', 'Horiz_Arrow_DN', 'Horiz_Arrow_Rllvr']
phase_3.5/models/gui/friendslist_gui.bam
['Horiz_Arrow_UP', 'Horiz_Arrow_DN', 'Horiz_Arrow_Rllvr']
phase_3.5/models/gui/friendslist_gui.bam
['FndsLst_ScrollUp', 'FndsLst_ScrollDN', 'FndsLst_ScrollUp_Rllvr']
phase_3.5/models/gui/friendslist_gui.bam
['FndsLst_ScrollUp', 'FndsLst_ScrollDN', 'FndsLst_ScrollUp_Rllvr']
phase_3.5/models/gui/friendslist_gui.bam
['Horiz_Arrow_UP', 'Horiz_Arrow_DN', 'Horiz_Arrow_Rllvr']
phase_3.5/models/gui/friendslist_gui.bam
['Horiz_Arrow_UP', 'Horiz_Arrow_DN', 'Horiz_Arrow_Rllvr']
phase_3.5/models/gui/friendslist_gui.bam
['Horiz_Arrow_UP', 'Horiz_Arrow_DN', 'Horiz_Arrow_Rllvr']
phase_3.5/models/gui/friendslist_gui.bam
['Horiz_Arrow_UP', 'Horiz_Arrow_DN', 'Horiz_Arrow_Rllvr']
phase_3/models/gui/quit_button.bam
['QuitBtn_UP', 'QuitBtn_DN', 'QuitBtn_RLVR']
phase_3/models/gui/quit_button.bam
['QuitBtn_UP', 'QuitBtn_DN', 'QuitBtn_RLVR']
phase_3/models/gui/quit_button.bam
['QuitBtn_UP', 'QuitBtn_DN', 'QuitBtn_RLVR']
phase_3/models/gui/quit_button.bam
['QuitBtn_UP', 'QuitBtn_DN', 'QuitBtn_RLVR']
test2
{401000001: <toontown.distributed.ToontownDistrict.ToontownDistrict instance at 0x1E486300>, 403000001: <toontown.distributed.ToontownDistrict.ToontownDistrict instance at 0x1E483328>, 402000001: <toontown.distributed.ToontownDistrict.ToontownDistrict instance at 0x1E765EE0>}
__handleAvatarChooserDone: 100000013, 'Little Zippy', ('rsl', 'ls', 'l', 'm', 19, 0, 19, 19, 10, 9, 0, 9, 7, 0), 1
2
Reading DNA file... phase_4/dna/storage.pdna
Reading DNA file... phase_3.5/dna/storage_interior.pdna
Reading DNA file... phase_6/dna/storage_DD.pdna
Reading DNA file... phase_6/dna/storage_DD_sz.pdna
Reading DNA file... phase_6/dna/donalds_dock_sz.pdna
Doing building quest
starting
frame rate: 29.6 fps|0.009d|1 avs||341|344||0x1002|0x6738|374.2MB|393.3MB|16330.4MB|450|nt.2.6.2|3.300,3.300|3,6 cpus|OpenGL
frame rate: 29.4 fps|0.009d|1 avs||641|644||0x1002|0x6738|374.4MB|393.3MB|16330.4MB|450|nt.2.6.2|3.300,3.300|3,6 cpus|OpenGL
Exception occurred while processing Func-updateQuestGui-22 of Sequence-31:
Sequence-31:
  0.000 Sequence-31 {
  0.000   *Func-updateQuestGui-22
  0.000   Wait dur 0.5
  0.500 }

Traceback (most recent call last):
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\showbase\ShowBase.py", line 1801, in __ivalLoop
    IntervalManager.ivalMgr.step()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\IntervalManager.py", line 96, in step
    self.__doPythonCallbacks()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\IntervalManager.py", line 124, in __doPythonCallbacks
    self.ivals[index].privPostEvent()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\MetaInterval.py", line 518, in privPostEvent
    self.__doPythonCallbacks()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\MetaInterval.py", line 494, in __doPythonCallbacks
    ival.privDoEvent(t, eventType)
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\Interval.py", line 207, in privDoEvent
    self.privInstant()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\FunctionInterval.py", line 88, in privInstant
    self.function(*self.extraArgs, **self.kw)
  File "<string>", line 161, in updateQuestGui
AttributeError: ToonBase instance has no attribute 'localAvatar'
Traceback (most recent call last):
  File "otp\otpbase\OTPBase.py", line 236, in run
    taskMgr.run()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\task\Task.py", line 502, in run
    self.step()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\task\Task.py", line 460, in step
    self.mgr.poll()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\showbase\ShowBase.py", line 1801, in __ivalLoop
    IntervalManager.ivalMgr.step()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\IntervalManager.py", line 96, in step
    self.__doPythonCallbacks()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\IntervalManager.py", line 124, in __doPythonCallbacks
    self.ivals[index].privPostEvent()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\MetaInterval.py", line 518, in privPostEvent
    self.__doPythonCallbacks()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\MetaInterval.py", line 494, in __doPythonCallbacks
    ival.privDoEvent(t, eventType)
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\Interval.py", line 207, in privDoEvent
    self.privInstant()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\FunctionInterval.py", line 88, in privInstant
    self.function(*self.extraArgs, **self.kw)
  File "<string>", line 161, in updateQuestGui
AttributeError: ToonBase instance has no attribute 'localAvatar'
